.github/workflows/gha-cache-push.yml

```diff
@@ -90,3 +92,24 @@ jobs:
 
           aws s3 cp ${{ env.BAZEL_CACHE_DEVCONTAINER_TAR }} ${{ env.S3_BUCKET_PATH }}/${{ env.BAZEL_CACHE_DEVCONTAINER_TAR}}
           aws s3 cp ${{ env.BAZEL_CACHE_REPO_DEVCONTAINER_TAR }} ${{ env.S3_BUCKET_PATH }}/${{ env.BAZEL_CACHE_REPO_DEVCONTAINER_TAR}}
+
+  virtualbox-push-cache:
+    runs-on: ubuntu-latest
```

Comment:
for the trigger of this job we could only use workflow_dispatch ,add`if : github.event.workflow_run.event == 'workflow_dispatch` for the job to not be triggered on other triggers.

Later we could do a path filter detecting vm version change in the lte/gateway/vagrantfile